src/test/jmespath/syntax.json

```diff
@@ -414,11 +414,6 @@
         "expression": "a{foo: 0}",
         "error": "syntax"
       },
-      {
```

Comment:
This is not an error case anymore. It has been moved https://github.com/adobe/json-formula/pull/22/commits/f928823ab186864eb08da26f9d3c72b7b255a0b5#diff-247931607bb8045c1f5499881916139eb8e02ccc27105b6209a0b397b6f6e907R438